extensions/vscode/src/daemon/dart-frog-daemon.ts

```diff
@@ -11,6 +11,26 @@ import {
 } from "./protocol";
 import { EventEmitter } from "events";
 
+/**
+ * An error that is thrown when the Dart Frog Daemon has not yet been invoked
+ * but a request is made to it.
+ */
+export class DartFrogDaemonWaiveError extends Error {
```

Comment:
Why the word Waive?

Reply:
I thought it was a nice word to describe that the daemon is, for the time being, refusing to do what is being told. From Oxford languages: "[refrain](https://www.google.com/search?sca_esv=557804163&rlz=1C5GCEM_en&q=refrain&si=ACFMAn_otZSKbpzAqD_RvWk4YSL-kVW3Sj_VIUhTuz6xsY30VXZifcUHNDC9PweUSsXRvkaTjHeC2Oibn1iTEG44J3_J8zBL-g%3D%3D&expnd=1) from [insisting](https://www.google.com/search?sca_esv=557804163&rlz=1C5GCEM_en&q=insisting&si=ACFMAn8hzZSJQsgXIYlkGc-z1vmpZXEEwIyWF97WcecY8vlxxQ_T-g_7zh_JADWdmorVSkok0JyBLXgQ-XWJfz9S7BgGVz4uGQ%3D%3D&expnd=1) on or using (a right or claim)."